packages/strapi-plugin-content-manager/admin/src/components/DraggedFieldWithPreview/DynamicComponent.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { withRouter } from 'react-router-dom';
```

Comment:
Instead of using a HoC we could use a hook:

`import { useHistory } from 'react-router-dom`